win32ss/gdi/eng/lineto.c

```diff
@@ -143,7 +143,7 @@ NWtoSE(SURFOBJ* OutputObj, CLIPOBJ* Clip,
             if (deltax < deltay)
```

Comment:
`HandleStyles()` uses `if (deltax > deltay)`,
while these functions use `if (deltax < deltay)`,
I wonder whether that is fine for the `==` case? (Fix or add an explaining code comment...)